data/ext/pending/issue-2935.ttl

```diff
@@ -0,0 +1,20 @@
+@prefix : <https://schema.org/> . 
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:pronouns a rdf:Property ;
+    rdfs:label "pronouns" ;
+    :domainIncludes :Person ; 
+    :isPartOf <https://pending.schema.org> ;
+    :rangeIncludes :Text,
+      :StructuredValue,
+      :DefinedTerm; 
+    :source <https://github.com/schemaorg/schemaorg/issues/2935> ;
+    rdfs:comment """A short string listing or describing pronouns for a person. Typically the person concerned is the best authority as pronouns are a critical part of personal identity and expression. Publishers and consumers of this information are reminded to treat this data responsibly, take country-specific laws related to gender expression into account, and be wary of out-of-date data and drawing unwarranted inferences about the person being described.
+    
+In English formulations such as "they/them", "she/her", and "he/him" are commonly used online and can also be used here. We do not intend to enumerate all possible micro-syntaxes in all languages. More structured and well-defined external values for pronouns can be referenced using the [[StructuredValue]] or [[DefinedTerm]] values.
```

Comment:
Some reference material considers such commas as this optional, but leaving them out often leads to confusing reading, as here, where "In formulations such as ... are" makes no sense and leads to a need to re-read as "formulations such as ... are".
```suggestion
In English, formulations such as "they/them", "she/her", and "he/him" are commonly used online and can also be used here. We do not intend to enumerate all possible micro-syntaxes in all languages. More structured and well-defined external values for pronouns can be referenced using the [[StructuredValue]] or [[DefinedTerm]] values.
```